packages/driver/cypress/integration/commands/navigation_spec.js

```diff
@@ -2285,7 +2284,7 @@ describe('src/cy/commands/navigation', () => {
       })
     })
 
-    it('waits for stability at the end of the command queue when not stable', (done) => {
+    it('waits for stability at the end of the command queue when not stable', { experimentalSessionAndOrigin: false }, (done) => {
```

Comment:
I would have expected this test to be throwing an error since the configuration option above described that it can't be updated at run-time. Is there a hack in place to test this or should this be a system test?

Reply:
This test functions because the logic being tested exists in the driver Setting the feature flag to false here _partially_ disables the feature, but that is enough for us to test what this test is trying to accomplish.